shub/image/init.py

```diff
@@ -95,6 +97,14 @@ def cli(project, base_image, base_deps, add_deps, requirements):
     if os.path.exists(dockefile_path):
         raise shub_exceptions.ShubException('Found a Dockerfile in the project directory, aborting')
     settings_module = scrapy_config.get('settings', project)
+    with utils.remember_cwd():
+        closest = closest_file('scrapy.cfg')
+        os.chdir(os.path.dirname(closest))
+        if not os.path.exists('setup.py'):
+            click.echo("Created setup.py for settings in {}"
+                       .format(settings_module))
+            _create_default_setup_py(settings=settings_module)
```

Comment:
Maybe the entire code block can be moved to a `_create_default_setup_py`? It seems like all this code (find closest scrapy.cfg, chdir, check existence) is duplicated in shub.deploy